### PR TITLE
Huggingface transformers integration test

### PR DESCRIPTION
**Pull request by HamidShojanazeri**
## Description

Add test integration for Huggingface transformers examples.

Fixes #(issue)

## Type of change
Added integration tests.
Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Feature/Issue validation/testing

Please describe the tests [UT/IT] that you ran to verify your changes and relevent result summary. Provide instructions so it can be reproduced. 
Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

- UT/IT execution results

- Logs

## Checklist:

- [X] Have you added tests that prove your fix is effective or that this feature works?
- [ ] New and existing unit tests pass locally with these changes?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

File: test/regression_tests.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ install_torchserve_from_source() {
   echo "Installing torchserve torch-model-archiver from source"
   ./scripts/install_from_src_ubuntu
   echo "Torchserve Succesfully installed"
-  
+  pip install transformers
```
Comment (dhanainme) [inline test/regression_tests.sh:32]: Can you please chain this lib installation to a previous pip install command.

https://github.com/HamidShojanazeri/serve/blob/b2e54c44ed369eb299a08079bbba406715458873/test/regression_tests.sh#L18

Comment (dhanainme) [reply test/regression_tests.sh:32]: This looks good to me otherwise.

Comment (HamidShojanazeri) [reply test/regression_tests.sh]: Thanks a lot @dhanainme, it's modified as suggested.

**Comment by dhanainme** [comment]
Verified to be working in the Nightly test run. Logs - https://torchserve-regression-test.s3.amazonaws.com/public/latest/test_exec.log

```

Iteration 12/14
 
→ Model Zoo - Register Model
  POST http://localhost:8081/models?url=https://torchserve.s3.amazonaws.com/mar_files/bert_seq_classification.mar&model_name=bert_seq_classification&initial_workers=1&synchronous=true [200 OK, 365B, 14s]
 
  prepare   wait    dns-lookup   tcp-handshake   transfer-start   download   process   total
  1ms       264µs   (cache)      (cache)         14s              2ms        40µs      14s  
 
  ✓  Successful POST request
 
→ Model Zoo - Inference Model
  POST http://localhost:8080/predictions/bert_seq_classification [200 OK, 241B, 87ms]
 
  prepare   wait    dns-lookup   tcp-handshake   transfer-start   download   process   total
  2ms       289µs   (cache)      (cache)         85ms             1ms        41µs      89ms 
 
  ✓  Successful POST request
 
Iteration 13/14
 
→ Model Zoo - Register Model
  POST http://localhost:8081/models?url=https://torchserve.s3.amazonaws.com/mar_files/bert_token_classification.mar&model_name=bert_token_classification&initial_workers=1&synchronous=true [200 OK, 368B, 15.7s]
 
  prepare   wait    dns-lookup   tcp-handshake   transfer-start   download   process   total
  1ms       268µs   (cache)      (cache)         15.7s            1ms        39µs      15.7s
 
  ✓  Successful POST request
 
→ Model Zoo - Inference Model
  POST http://localhost:8080/predictions/bert_token_classification [200 OK, 731B, 73ms]
 
  prepare   wait    dns-lookup   tcp-handshake   transfer-start   download   process   total
  2ms       261µs   (cache)      (cache)         70ms             1ms        42µs      74ms 
 
  ✓  Successful POST request
 
Iteration 14/14
 
→ Model Zoo - Register Model
  POST http://localhost:8081/models?url=https://torchserve.s3.amazonaws.com/mar_files/roberta_qa.mar&model_name=roberta_qa&initial_workers=1&synchronous=true [200 OK, 352B, 15.3s]
 
  prepare   wait    dns-lookup   tcp-handshake   transfer-start   download   process   total
  1ms       276µs   (cache)      (cache)         15.3s            1ms        40µs      15.3s
 
  ✓  Successful POST request
 
→ Model Zoo - Inference Model
  POST http://localhost:8080/predictions/roberta_qa [200 OK, 280B, 84ms]
 
  prepare   wait    dns-lookup   tcp-handshake   transfer-start   download   process   total
  2ms       271µs   (cache)      (cache)         80ms             2ms        41µs      85ms 
 
  ✓  Successful POST request
```

**Pull request merged**